### PR TITLE
Fix qemu CVE-2023-3354

### PR DESCRIPTION
**Pull request by fintelia**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

Fix for qemu CVE-2023-3354

###### Change Log  

- Fix for qemu CVE-2023-3354

###### Does this affect the toolchain?  

**NO**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2023-3354

###### Test Methodology

- Pipeline build id: 443045

### CODE REVIEW

File: SPECS/qemu/qemu.spec [hunk 1]
```diff
@@ -274,6 +274,7 @@ Patch1013:      CVE-2022-3165.patch
 # https://gitlab.com/qemu-project/qemu/-/issues/541
 Patch1014:      CVE-2021-3750.patch
 Patch1015:      CVE-2022-36648.patch
+Patch1016:      CVE-2023-3354.patch
```
Comment (neha170) [inline SPECS/qemu/qemu.spec:277]: Please bump up the release and add a line to the changelog, thanks

**Comment by PawelWMS** [comment]
@fintelia, just a heads-up: all PRs against `fasttrack/2.0` should be rebased, since we finished a monthly update, which also updated `fasttrack/2.0`. Rebasing (or merging) the latest state of the branch will re-trigger the PR check and allow us to validate that the latest state of the repo still builds correctly with the changes from the PRs.

**Comment by aditjha-msft** [comment]
@PawelWMS I merged the latest fasttrack/2.0 into this fork, I think the PR check failed on the junit_xml that I see you have a draft PR open for. Is this PR okay to merge for the qemu cve?

**Comment by fintelia** [comment]
If the relevant pipelines are now passing, then either this and/or #6634 should be OK to merge. Though you should probably first get confirmation about whether the CVE should still be fast-tracked

**Comment by PawelWMS** [comment]
> @PawelWMS Pawel Winogrodzki FTE I merged the latest fasttrack/2.0 into this fork, I think the PR check failed on the junit_xml that I see you have a draft PR open for. Is this PR okay to merge for the qemu cve?

Sorry for the late reply - I missed your question.

We should get the PR pipeline under control soon, but in the meantime, if it fails to install the Python module to parse the test results, you can still check the tests manually by opening the logs from the pipeline artifacts and looking for any logs with `FAILED_TEST` in their name. And then you can use the logs to debug and fix the tests.

**Comment by PawelWMS** [comment]
Overriding check - the failing ptests for `libguestfs` and nbdkit` are not regressions and are too complex to fix as part of this PR. Merging.

**Pull request merged**